proto/batch.proto

```diff
@@ -44,6 +48,7 @@ message Batch {
     repeated SwapKey keys = 3;
     repeated SwapKey multi_swaps_keys = 4;
     repeated MultiSwap multi_swaps   = 5;
+    repeated pendingTx pendings = 6; // payload for field txIDs
```

Comment:
Why the message `pendingTx` starts from lower case?

Reply:
that's legacy

Reply:
fix